softwareComponents/torqueComputation/src/joint.cpp

```diff
@@ -0,0 +1,212 @@
+#include "joint.hpp"
```

Comment:
Čekal bych že se budeš držet toho, že .hpp soubory jsou v `include/` a .cpp v `src/`

Reply:
Ano, je to záměr - uživatel ty vvnitřní soubory nepotřebuje.